frontend/trpc/routes/paymentManagement.ts

```
@@ -0,0 +1,280 @@
+    .input(z.object({
+      companyId: z.string(),
+      dividendRoundId: z.number(),
+    }))
+    .query(async ({ input, ctx }) => {
+      try {
+        const response = await fetch(
+          `${process.env.BACKEND_URL}/internal/companies/${ctx.company!.externalId}/dividend_rounds/${input.dividendRoundId}/payment_status`,
+          {
+            method: "GET",
+            headers: {
+              "Authorization": `Bearer ${ctx.headers.authorization?.replace("Bearer ", "")}`,
```

ESLint

[error] Replace `"Authorization"` with `Authorization`

(prettier/prettier)